tutorials/xr/setting_up_xr.rst

```diff
@@ -89,6 +90,24 @@ Next we need to add a script to our root node. Add the following code into this
 		else:
 			print("OpenXR not initialised, please check if your headset is connected")
 
+  .. code-tab:: csharp
+
+    public partial class XRInitializer : Node3D
+    {
+        public override void _Ready()
+        {
+            XRInterface xrInterface = XRServer.FindInterface("OpenXR");
+            if (xrInterface != null && xrInterface.IsInitialized())
+            {
+                GetViewport().UseXR = true;
+
+                GD.Print("OpenXR Initialized Successfully!");
```

Comment:
The GDScript example has these lines in a different order. We should try to be consistent.

```suggestion
                GD.Print("OpenXR initialized successfully");

                GetViewport().UseXR = true;
```